admin/class-metabox-addon-section.php

```diff
@@ -0,0 +1,39 @@
+<?php
+/**
+ * @package WPSEO\Admin
+ */
+
+/**
+ * Generates and displays a section containing metabox tabs that have been added by other plugins through the
+ * `wpseo_tab_header` and `wpseo_tab_content` actions.
+ *
+ * Class WPSEO_Metabox_Addon_Section
```

Comment:
What exactly is the use of this lines? This can only get stale, let's remove it.